doc/sphinx/language/extensions/evars.rst

```diff
@@ -44,12 +44,14 @@ existential variable is represented by “?” followed by an identifier.
 
 In the general case, when an existential variable :n:`?@ident` appears
 outside of its context of definition, its instance, written under the
-form :n:`{ {*; @ident := @term} }` is appending to its name, indicating
+form :n:`{ {*; @ident := @term} }`, is appended to its name, indicating
 how the variables of its defining context are instantiated.
-The variables of the context of the existential variables which are
-instantiated by themselves are not written, unless the :flag:`Printing Existential Instances` flag
```

Comment:
Is the part about `Printing Existential Instances` still relevant? I agree that the text is a bit cryptic here. Perhaps just a "See also" is in order.

Reply:
It is relevant, but I agree that the place of the example is wrong: it should be in the part describing the `Printing Existential Instances` flag. I’ve just pushed a commit with the change.

Reply:
About the relevance:
```coq
Check (fun x y => _) 0 1. (* (fun x y : nat => ?y) 0 1 : ?T@{x:=0; y:=1} *)
Set Printing Existential Instances.
Check (fun x y => _) 0 1. (* (fun x y : nat => ?y@{x:=x; y:=y}) 0 1 : ?T@{x:=0; y:=1} *)
```